### ISSUE TITLE
Abonnieren Button hat eine andere Schriftart

### ISSUE THREAD

**Issue by wintermeyer** [wintermeyer/anfaengerkrebs.de#4]
Der Button-Text hat eine andere Schriftart, als der Rest der Seite. Der Text ist auch einen Tick zu klein.

@xamiam: Wo muss ich da suchen?

<img width="294" alt="Bildschirmfoto 2020-01-15 um 07 41 52" src="https://user-images.githubusercontent.com/69770/72411151-925f4680-376a-11ea-96a1-fdc3b1397f84.png">

**Comment by xamiam** [comment]
Ist das noch relevant? Der Button ist in HTML ausgeblendet...

**Comment by wintermeyer** [comment]
> Ist das noch relevant? Der Button ist in HTML ausgeblendet...

Ja, ist noch relevant. Der Feed wird gerade bei Apple überprüft. Bis dahin ist der Button nicht aktiv.
